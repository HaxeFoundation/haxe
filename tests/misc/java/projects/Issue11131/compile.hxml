-cp src
-main Main
--jvm run.jar
--cmd java -jar run.jar