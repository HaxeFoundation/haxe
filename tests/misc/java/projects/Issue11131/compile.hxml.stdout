6
9
6
9